strawberryfields/gbs/data.py

```diff
@@ -408,13 +422,67 @@ class Mutag3(Dataset):
 
     Attributes:
         n_mean = 6
-        n_max = 20
         threshold = False
         n_samples = 20000
         modes = 19
     """
 
     _data_filename = "MUTAG_3"
     n_mean = 6
-    n_max = 20
     threshold = False
+
+
+# pylint: disable=abstract-method
+class MoleculeDataset(Dataset, ABC):
+    r"""Class for loading datasets of pre-generated samples from molecules.
+
+    Attributes:
+        w (array): normal mode frequencies of the electronic ground state (:math:`\mbox{cm}^{-1}`)
+        wp (array): normal mode frequencies of the electronic excited state (:math:`\mbox{cm}^{-1}`)
+        Ud (array): Duschinsky matrix
+        delta (array): Duschinsky displacement vector

```

Comment:
```suggestion
        delta (array): displacement vector
```
As far as I understand, the Duschinsky displacement is the vector d that defines the transformation between mode coordinates q'= Uq+d. The delta we use here isn't quite that, but a rescaled version

Reply:
True, delta is a unitless version of d.